### PR TITLE
respect schema for all queries

### PR DESCRIPTION
**Pull request by wwitzel3**
Signed-off-by: Wayne Witzel III <wayne@stacklet.io>

### CODE REVIEW

File: migrations/versions/969126bd800f_.py [hunk 1]
```diff
@@ -21,18 +21,21 @@
...
+    dashboard_result = db.session.execute(f"SELECT id, layout FROM {dashboards}")
     for dashboard in dashboard_result:
         print("  Updating dashboard: {}".format(dashboard["id"]))
         layout = simplejson.loads(dashboard["layout"])
 
         print("    Building widgets map:")
         widgets = {}
         widget_result = db.session.execute(
-            "SELECT id, options, width FROM widgets WHERE dashboard_id=:dashboard_id",
+            f"SELECT id, options, width FROM {widgets} WHERE dashboard_id=:dashboard_id",
```
Comment (kapilt) [inline migrations/versions/969126bd800f_.py:38]: hmmm.. we should be able to set the default schema on the connection to be less invasive? also why do we care about historical upgrades. they do a stamp with current afaiks

Comment (wwitzel3) [reply migrations/versions/969126bd800f_.py:38]: I was being thorough, but it probably isn't strictly needed on the migrations.
As for setting is on the connection, that doesn't work here because these are executing the queries directly, they don't go through a translation map like if it was using the `Table` object to generate the select.

Comment (wwitzel3) [reply migrations/versions/969126bd800f_.py:38]: @kapilt I could modify the direct execute queries to use the `Table` objects to generate the select statements and then the schema on the connection engine would work.

File: redash/monitor.py [hunk 2]
```diff
@@ -30,15 +30,23 @@ def get_object_counts():
...
+    query_results = "query_results"
+    if schema:
+        query_results = ".".join([schema, query_results])
+
     database_metrics = []
     queries = [
         [
             "Query Results Size",
-            "select pg_total_relation_size('query_results') as size from (select 1) as a",
+            f"select pg_total_relation_size('{query_results}') as size from (select 1) as a",
```
Comment (wwitzel3) [inline redash/monitor.py:49]: This doesn't respect the schema on engine since it is a nested closure of the `pg_total_relation_size` of the query. This is the only place I had to manually prefix.

**Review by kapilt — APPROVED** [review]
lgtm, albeit there's a few changes here unrelated to schema respect

File: client/app/lib/utils.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ export function localizeTime(time) {
     .utc()
     .hour(hrs)
     .minute(mins)
-    .local()
+    .locale()
```
Comment (wwitzel3) [inline client/app/lib/utils.js:54]: Drive-by to fix a frontend error discovered when making the system stats page work. Want in a different PR?

Comment (wwitzel3) [reply client/app/lib/utils.js:54]: @kapilt I removed the drive-by formatting, would you prefer this as a distinct PR?

Comment (wwitzel3) [reply client/app/lib/utils.js:54]: I'll put this in a different PR because it will be easier to submit the schema changes upstream that way.

Comment (kapilt) [reply client/app/lib/utils.js]: your call, sounds good.
